libr/util/str.c

```diff
@@ -1351,6 +1351,36 @@ R_API char *r_str_escape(const char *buf) {
 	return r_str_escape_ (buf, false, true, true, false, true);
 }
 
+// Return MUST BE surrounded by double-quotes
+R_API char *r_str_escape_sh(const char *buf) {
+	r_return_val_if_fail (buf, NULL);
+	char *new_buf = malloc (1 + strlen (buf) * 2);
+	if (!new_buf) {
+		return NULL;
+	}
+	const char *p = buf;
+	char *q = new_buf;
+	while (*p) {
+		switch (*p) {
+#if __WINDOWS__
```

Comment:
so, in case of WINDOWS, a string like `hello "world"` becomes `hello "\"world"\"`? Is that expected? BTW, it would be awesome if you could as well add a short unit test for this in `unit/test/test_str.c`, if that helps you checking the output of this function.